.github/workflows/integration-test.yml

```
@@ -0,0 +1,94 @@
+name: Integration Tests
+
+on:
+  workflow_call:
+    inputs:
+      image-ref:
+        required: true
+        type: string
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-24.04
+    env:
+      SSH_PORT: 2222
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build QCOW Image
+        id: build
+        uses: bootc-warehouse/bootc-image-builder-action@main
```

```suggestion
        uses: osbuild/bootc-image-builder-action@v0.0.2
```

This action has been moved upstream, and releases have been created